app/styles/modules/_user_info.scss

```diff
@@ -0,0 +1,86 @@
+#user-info {
+  height: $default-grid + $default-gap;
+  flex: 0 0 $default-grid + $default-gap;
+
+  .trigger {
+    border-radius: $double-radius;
+    cursor: pointer;
+    height: 100%;
+    transition: background $default;
+    width: 100%;
+
+
+    .mask {
+      border: 1px solid lighten($satan-grey, 10);
+      border-radius: $round-radius;
+      height: $default-grid;
+      overflow: hidden;
+      width: $default-grid;
+    }
+  }
+
+  .menu {
+    @include color-scheme($middle-grey, $white);
+    border: 0.5px solid lighten($satan-grey, 20);
+    border-radius: $double-radius;
+    box-shadow: 0 5px 5px $translucent-black;
+    font-size: $small-font-size;
+    margin: $small-gap 0 0 -5px;
+    position: absolute;
+    width: $default-grid * 8;
+
+// leaving fussy caret border/position CSS unvariablized for clarity
+    &:before {
+      border-bottom: 9px solid lighten($satan-grey, 20);
+      border-left: 8px solid transparent;
+      border-right: 8px solid transparent;
+      content: '';
+      height: 0;
+      left: $half-grid + 1;
+      position: absolute;
+      top: -10.5px;
+      width: 0;
+    }
+
+    &:after {
+      border-bottom: 10px solid $white;
+      border-left: 9px solid transparent;
+      border-right: 9px solid transparent;
+      content: '';
+      height: 0;
+      left: $half-grid;
+      position: absolute;
+      top: -9.5px;
+      width: 0;
+    }
+  }
+
+  .menu-chunk {
+    padding: $default-font-size;
+
+    h4 {
+      margin: 0 0 $small-gap;
+    }
+
+    ul {
+      list-style: none;
+      margin: 0;
+      padding: 0;
+    }
+
+    a {
+      color: $dark-grey;
+      display: block;
+      padding: $default-gap 0 0 0;
+      text-decoration: none;
+      
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+
+    + .chunk {
```

Comment:
Me thinks `.chunk` should be `.menu-chunk` here.
Changing that gives me:
![chronicle](https://cloud.githubusercontent.com/assets/557895/6066867/3ad8503e-ad25-11e4-86de-75591eacc922.png)